### ISSUE TITLE
Crashes on startup

### ISSUE THREAD

**Issue by richardmitic** [sannies/isoviewer#1]
The JavaFX releases (1.0.3 and 1.0.4) crash when starting.
Windows 7
JDK 1.8.0_31

Here's the backtrace:
```
Exception in Application start method
java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.lang.reflect.Method.invoke(Unknown Source)
        at com.sun.javafx.application.LauncherImpl.launchApplicationWithArgs(Unknown Source)
        at com.sun.javafx.application.LauncherImpl.launchApplication(Unknown Source)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.lang.reflect.Method.invoke(Unknown Source)
        at sun.launcher.LauncherHelper$FXHelper.main(Unknown Source)
Caused by: java.lang.RuntimeException: Exception in Application start method
        at com.sun.javafx.application.LauncherImpl.launchApplication1(Unknown Source)
        at com.sun.javafx.application.LauncherImpl.lambda$launchApplication$147(Unknown Source)
        at com.sun.javafx.application.LauncherImpl$$Lambda$48/752848266.run(Unknown Source)
        at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.RuntimeException: A cast to int has gone wrong. Please contact the mp4parser discussion group (4294967096)
        at com.googlecode.mp4parser.util.CastUtils.l2i(CastUtils.java:30)
        at com.googlecode.mp4parser.authoring.Mp4TrackImpl.<init>(Mp4TrackImpl.java:149)
        at com.github.sannies.isoviewer.IsoViewerFx.openFile(IsoViewerFx.java:80)
        at com.github.sannies.isoviewer.IsoViewerFx.loadPosAndSize(IsoViewerFx.java:216)
        at com.github.sannies.isoviewer.IsoViewerFx.start(IsoViewerFx.java:111)
        at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$153(Unknown Source)
        at com.sun.javafx.application.LauncherImpl$$Lambda$51/1013840123.run(Unknown Source)
        at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$166(Unknown Source)
        at com.sun.javafx.application.PlatformImpl$$Lambda$45/584634336.run(Unknown Source)
        at com.sun.javafx.application.PlatformImpl.lambda$null$164(Unknown Source)
        at com.sun.javafx.application.PlatformImpl$$Lambda$47/1145228818.run(Unknown Source)
        at java.security.AccessController.doPrivileged(Native Method)
        at com.sun.javafx.application.PlatformImpl.lambda$runLater$165(Unknown Source)
        at com.sun.javafx.application.PlatformImpl$$Lambda$46/501263526.run(Unknown Source)
        at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(Unknown Source)
        at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
        at com.sun.glass.ui.win.WinApplication.lambda$null$141(Unknown Source)
        at com.sun.glass.ui.win.WinApplication$$Lambda$37/96639997.run(Unknown Source)
        ... 1 more
Exception running application com.github.sannies.isoviewer.IsoViewerFx
```

**Comment by sannies** [comment]
Ok. It fails opening a file on start up.  That also happened to me a few times and I solved it by deleting it. Do you remember which file it was?
I'll make it more robust later.

**Comment by richardmitic** [comment]
Actually if happens both if you try to open it with a file (```java -jar isoviewer.jar movie.mp4```) or just open it raw (```java -jar isoviewer.jar```). The exception is the same both times.

**Comment by sannies** [comment]
You can no longer open a file by adding it to the command line. This is why you still get the same error. I released 1.0.5 - it should be more robust. Give it a try!

**Comment by richardmitic** [comment]
Awesome! It works perfectly now.
Thanks for all the good work. I use this pretty much every day.
